docs/funcotator/forum_info/forum_post_tutorial.md

```diff
@@ -112,19 +114,73 @@ Versioned gzip archives of data source files are provided here:
 
 <a name="1.1.2.2"></a>
 ### 1.1.2.2 - gnomAD
-The pre-packaged data sources include gnomAD, a large database of known variants.  gnomAD is split into two parts - one based on exome data, one based on whole genome data.
+The pre-packaged data sources include a subset gnomAD, a large database of known variants.  This subset contains a set of INFO fields, primarily containing allele frequency data.  gnomAD is split into two parts - one based on exome data, one based on whole genome data.  These two data sources are not equivalent and for complete coverage using gnomAD, we recommend annotating with both.
```

Comment:
subset *of* gnomAD

Reply:
Specify that the subset of INFO fields is much smaller than the original vcf.  It is, right?

Reply:
Good catch! 

Yes it's the info fields that were reduced in size.